ldes-server-port-admin-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/AdminEventStreamsRestController.java

```diff
@@ -35,24 +43,34 @@ public AdminEventStreamsRestController(EventStreamService eventStreamService, Sh
 		this.eventStreamValidator = eventStreamValidator;
 	}
 
-	@InitBinder
-	private void initBinder(WebDataBinder binder) {
-		binder.setValidator(eventStreamValidator);
-	}
-
 	@GetMapping
-	public List<EventStreamResponse> getEventStreams() {
-		return eventStreamService.retrieveAllEventStreams().stream().map(eventStream -> {
-			List<ViewSpecification> views = viewService.getViewsByCollectionName(eventStream.getCollection());
-			ShaclShape shaclShape = shaclShapeService.retrieveShaclShape(eventStream.getCollection());
-			return new EventStreamResponse(eventStream.getCollection(), eventStream.getTimestampPath(),
-					eventStream.getVersionOfPath(), views, shaclShape.getModel());
-		}).toList();
+	@Operation(summary = "Retrieve list of configured Event Streams")
+	@ApiResponse(responseCode = "200", content = {
+			@Content(mediaType = NQUADS),
+			@Content(mediaType = JSON_LD),
+			@Content(mediaType = TURTLE)
+	})
+	public String getEventStreams(@Parameter(hidden = true) @RequestHeader("Accept") String contentType) {
```

Comment:
The content type is being used in the method, however, every parameter in the GetMapping will be visualised in Swagger.
Whilst Swagger will already pass the content-type through the Accept header on its own. Therefor this can be ignored ;)